crates/uv-settings/src/settings.rs

```diff
@@ -1062,6 +1062,61 @@ pub struct PipOptions {
         "#
     )]
     pub no_extra: Option<Vec<ExtraName>>,
+    /// Include optional dependencies from the specified group; may be provided more than once.
+    ///
+    /// Only applies to `pyproject.toml`, `setup.py`, and `setup.cfg` sources.
+    #[option(
+        default = "[]",
+        value_type = "list[str]",
+        example = r#"
+            group = ["dev", "docs"]
+        "#
+    )]
+    pub group: Option<Vec<GroupName>>,
```

Comment:
Whether these were supposed to pass through the shared PipOptions and not a side-thing was an uncertainty for me. Overall I assumed they're equivalentish to the `extra` settings and those are here, so, seems fine?

Reply:
Seems right to me, though the option abstractions are not my forte.